v3io/aio/dataplane/kv.py

```diff
@@ -228,6 +231,8 @@ async def scan(
             The container on which to operate.
         table_path (Required) : str
             The full path of the table
+        table_name (Optional) : str
+            The name the table - using GetItems TableName (if used, table_path must be an empty string). Allows for specifying all tables with '/*'
```

Comment:
I think something like this would be better:
```suggestion
            The name of the table. Allows for specifying all tables with '/*'. If used, table_path must be an empty
            string.
```
Since no other parameter mentions the underlying attribute, line is too long, and just for readability in general.